crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -0,0 +1,62 @@
+use std::path::PathBuf;
+
+use super::CommandBase;
+use crate::{
+    cli::DaemonCommand,
+    daemon::{DaemonConnector, DaemonError},
+};
+
+/// Runs the daemon command.
+pub async fn main(command: &DaemonCommand, base: &CommandBase) -> Result<(), DaemonError> {
+    let (dont_start, dont_kill) = match command {
+        DaemonCommand::Status { .. } => (true, true),
+        DaemonCommand::Restart | DaemonCommand::Stop => (true, false),
+        DaemonCommand::Start => (false, false),
+    };
+
+    let connector = DaemonConnector {
+        dont_start,
+        dont_kill,
+        pid_file: base.daemon_file_root().join("turbod.pid"),
+        sock_file: base.daemon_file_root().join("turbod.sock"),
+    };
+
+    let mut client = connector.connect().await?;
+
+    match command {
+        DaemonCommand::Restart => {
+            client.restart().await?;
+        }
+        DaemonCommand::Start => {} // no-op
```

Comment:
A comment explaining that `connector.connect()` will start the server would be useful as to why this is a noop.